api/sip/client.py

```diff
@@ -352,12 +352,12 @@ def make_secure_connection(self):
         tmp_ssl_cert_path = None
         tmp_ssl_key_path = None
         if self.ssl_cert:
-            fd, tmp_ssl_cert_path = tempfile.mkstemp()
-            os.write(fd, self.ssl_cert)
+            fd, tmp_ssl_cert_path = tempfile.mkstemp('wb')
+            os.write(fd, self.ssl_cert.encode("ascii"))
```

Comment:
AFAICT SSL certs are always supposed to be ASCII, but we're storing them in the database as Unicode.